src/components/DuosHeader.js

```diff
@@ -67,6 +67,72 @@ const styles = {
   }
 };
 
+export const headerTabsConfig = [
+  {
+    label: 'Admin Console',
+    link: '/admin_manage_dar_collections',
+    children: [
+      { label: 'DAR Requests', link: '/admin_manage_dar_collections' },
+      { label: 'Dataset Catalog', link: '/dataset_catalog' },
+      { label: 'DACs', link: '/manage_dac' },
+      { label: 'Statistics', link: '/summary_votes' },
+      { label: 'Users', link: '/admin_manage_users' },
+      { label: 'Institutions', link: '/admin_manage_institutions' },
+      { label: 'Library Cards', link: '/admin_manage_lc' }
+    ],
+    isRendered: (user) => user.isAdmin
+  },
+  {
+    label: 'SO Console',
+    link: '/signing_official_console/researchers',
+    children: [
+      { label: 'Researchers', link: '/signing_official_console/researchers' },
+      { label: 'DAR Requests', link: '/signing_official_console/dar_requests' }
+    ],
+    isRendered: (user) => user.isSigningOfficial
+  },
+  {
+    label: 'Researcher Console',
+    link: '/researcher_console',
+    search: 'researcher_console',
+    children: [
+      { label: 'DAR Requests', link: '/researcher_console' },
+      { label: 'Data Catalog', link: '/dataset_catalog' }
+    ],
+    isRendered: (user) => user.isResearcher
+  },
+  {
+    label: 'DAC Chair Console',
+    link: '/chair_console',
+    search: 'chair_console',
+    children: [
+      { label: 'DAR Requests', link: '/chair_console' },
+      { label: 'Datasets', link: '/dataset_catalog' },
+      { label: 'DAC Members', link: '/manage_dac' }
+    ],
+    isRendered: (user) => user.isChairPerson
+  },
+  {
+    label: 'DAC Member Console',
+    link: '/member_console',
+    search: 'member_console',
+    children: [
+      { label: 'DAR Requests', link: '/member_console' },
+      { label: 'Datasets', link: '/dataset_catalog' },
+    ],
+    isRendered: (user) => user.isMember
+  },
+  {
+    label: 'DS Console',
+    link: '/data_submission_form',
+    search: 'data_submission_form',
+    children: [
+      { label: 'Datasets', link: '/data_submission_form' }
+    ],
+    isRendered: (user) => user.isDataSubmitter
+  }
```

Comment:
I think we should move the researcher console tab config to the end of this array if we want Data Submitters to see the Data Submitter console first. Every user has a researcher role, so the `find` function that gets the first console would hit the researcher console first.

Reply:
gotcha, I agree! Moving the researcher block to the end of the array 👍